src/components/Introduction.jsx

```diff
@@ -328,7 +340,8 @@ const Introduction = () => {
               style={styles.button}
               onClick={() => {
                 if (
-                  treatment.viewType === ViewType.word &&
+                  (treatment.viewType === ViewType.word ||
+                    treatment.viewType === ViewType.calendarWord) &&
```

Comment:
this change might not be needed, re-review later

Reply:
That is, it is never clicked.  I can change it to be similar to the useEffect code that enables the button.

Reply:
I fixed the logic on main.  I removed the check if it is a word treatment. That isn't needed.  Be sure to merge my changes in from main.  I did a lot of little changes.